client/landing/jetpack-cloud/sidebar.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default function() {
+	return (
+		<div>
+			<ul>
+				<li>
+					<a href="/jetpack-cloud">Dashboard</a>
+				</li>
+				<li>
+					<a href="/jetpack-cloud/backups">Backups</a>
```

Comment:
Should we have custom config flags for `scan` and `backup`, and enable those sections only if those config flags are enabled?